kythe/proto/xref_serving.proto

```diff
@@ -55,7 +69,7 @@ message FileDecorations {
   // Columnar key: 10-start-end-kind-target
   message Target {
     // Starting offset of this file decoration.
-    int32 start_offest = 1;
+    int32 start_offset = 1;
```

Comment:
My diff view is saying something changed on this line, but I can't tell what it is.

Reply:
`offset` was misspelled.